lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -157,6 +157,16 @@
             - Purge any existing application settings. Replace web app application settings with app_settings.
         type: bool
 
+    power_action:
```

Comment:
just wonder if power_action is a proper name for this, because it's really a state.
I was just thinking about virtual machines, and a kind of I don't like the interface there either (started / restarted / allocated)

Reply:
updated to app_state per offline sync